landscaper/gardenlet/example/landscaper-resources/01-seed-cluster-target.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Target
+metadata:
+  name: seed-cluster
+  namespace: default
+spec:
+  type: landscaper.gardener.cloud/kubernetes-cluster
+  config:
+    kubeconfig: |
+      apiVersion:...
+          # here goes the kubeconfig of the runtime cluster
```

Can you leave 1 empty line at the end of the file?
```suggestion
          # here goes the kubeconfig of the runtime cluster

```